.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please don't commit this file! (and feel free to add "/.vscode/" to the top of the .gitignore file to prevent this from happening again)

Reply:
Sorry -- not sure why that appeared there, but I removed it.